src/types/models/carSales.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiSearchListing } from "../../types/models/listing"
+import { SearchMessageLead } from "../../types/models"
+
+export interface Buyer {
+  email: string
+  firstName: string
+  lastName: string
+  phone: string
+}
+
+export type FirstListingParams = Pick<
```

Comment:
- I think the fist second part approach is rather hard to understand as the consumer would have to custom combine the type.

What I had in mind was something like this
```
export interface CareSalesListing
  extends Pick<
    ApiSearchListing,
    | "id"
    | "firstRegistrationDate"
    | "price"
    | "mileage"
    | "externalListingId"
    | "make"
    | "model"
    | "type"
  > {
  image: string
}
```